.github/workflows/yehezgun.yml

```diff
@@ -0,0 +1,38 @@
+name: Template Git Actions
+
+run-name:
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  Base-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event"
```

Comment:
Do you think this is needed?

```suggestion

```

Reply:
Deleted, it's just for debugging purpose before